github/table_github_license.go

```diff
@@ -27,53 +26,67 @@ func tableGitHubLicense() *plugin.Table {
 		Columns: []*plugin.Column{
 
 			// Top columns
-			{Name: "spdx_id", Description: "The Software Package Data Exchange (SPDX) id of the license.", Type: proto.ColumnType_STRING, Transform: transform.FromField("SPDXID")},
+			{Name: "spdx_id", Description: "The Software Package Data Exchange (SPDX) id of the license.", Type: proto.ColumnType_STRING, Transform: transform.FromField("SpdxId")},
 			{Name: "name", Description: "The name of the license.", Type: proto.ColumnType_STRING},
-			{Name: "html_url", Description: "The HTML URL of the license.", Type: proto.ColumnType_STRING, Hydrate: tableGitHubLicenseGetData},
+			{Name: "html_url", Description: "The HTML URL of the license.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Url")},
```

Comment:
To stay in line with GitHub's API (like we do with other columns), should we rename this column to `url` instead?

Reply:
External urls appear to be named html_url in other tables also, I'd recommend leaving as such for now and reviewing as we change over other tables to v4 if this is something we can make a standard approach on?